DEVELOPMENT.md

```diff
@@ -0,0 +1,83 @@
+## Setup
+
+### Prerequisites
+
+  * [libgit2](https://libgit2.org/)
+    * macOS: `brew instal libgit2`
```

Comment:
note: I have had issues with versions of libgit2. kpack only seems to support v1.1.0

Reply:
Interesting. I've been running 1.1.1. What issues have you noticed?

Reply:
I meant 1.1.x. Ubuntu bionic for example ships with an older version of libgit2 which results in compilation errors.

Reply:
Ahh, I see. I'll add a note.